solr/contrib/build.gradle

```diff
@@ -0,0 +1,77 @@
+import groovy.json.JsonOutput
+import org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency
```

Comment:
unused?

Reply:
yeah forgot to remove all the debugging stuff.